src/courses/advanced/11.md

```diff
@@ -119,20 +122,22 @@ jobs:
         run: |
           curl -# -s -F data=@results/pipeline_run_attested.json -F "filename=${{ github.actor }}-pipeline-demo-${{ github.sha }}.json" -F "public=true" -F "evaluationTags=${{ github.repository }},${{ github.workflow }}" -H "Authorization: Api-Key ${{ secrets.HEIMDALL_API_KEY }}" "https://heimdall-demo.mitre.org/evaluations"
 
-      - name: VERIFY - Display our results summary 
+      - name: VERIFY - Display our results summary
         run: |
           saf view summary -i results/pipeline_run_attested.json
-      
+
       # check if the pipeline passes our defined threshold
-      - name: VERIFY - Ensure the scan meets our results threshold           
+      - name: VERIFY - Ensure the scan meets our results threshold
         run: |
           saf validate threshold -i results/pipeline_run_attested.json -F threshold.yml
 ```
+
 :::
 
 A few things to note here:
+
 - We added the `summary` step because it will print us a concise summary inside the pipeline job view itself. That command takes one file argument; the results file we want to summarize.
```

Comment:
```suggestion
- We added the `summary` step because it will print us a concise summary inside the pipeline job view itself. That command takes one file argument: the results file we want to summarize.
```